IssueComment
  { issueCommentUpdatedAt = 2014 (-10) (-08) 19 : 00 : 03 UTC
  , issueCommentUser =
      SimpleUser
        { simpleUserId = Id 7431361
        , simpleUserLogin = N "larkost"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/7431361?v=3"
        , simpleUserUrl = "https://api.github.com/users/larkost"
        , simpleUserType = OwnerUser
        }
  , issueCommentUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/comments/58409054"
  , issueCommentHtmlUrl =
      "https://github.com/rethinkdb/rethinkdb/issues/3151#issuecomment-58409054"
  , issueCommentCreatedAt = 2014 (-10) (-08) 19 : 00 : 03 UTC
  , issueCommentBody =
      "An issue came up in conversation this morning with @segphault, and I think the solutions could be related. He was working with someone that has multiple clusters of RethinkDB running at the same time on different virtual networks. It sounds like they had a problem with what computers were on what networks, and have IP address overlaps between the two. So when RethinkDB came up on one computer it was told to join to a computer that happened to be from the \"wrong\" cluster, and naturally started seeing (and backfilling) the \"wrong\" data.\r\n\r\nThis is all RethinkDB working as it is supposed to do, but if we did have a certificate (or something similar) associated with a cluster that all servers needed in order to connect, then this would allow sys-admins with multiple clusters to ensure that only the right machines joined the cluster."
  , issueCommentId = 58409054
  }